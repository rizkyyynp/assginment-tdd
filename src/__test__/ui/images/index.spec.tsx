import React from "react";
import {render } from "@testing-library/react";
import Images from "@/components/ui/image";

describe("render images", () => {
    it("should render images", () => {
        const { container } = render(<Images src="https://reqres.in/img/faces/7-image.jpg" alt="Photo of user" />);
        expect(container).toMatchSnapshot();
    });
});
