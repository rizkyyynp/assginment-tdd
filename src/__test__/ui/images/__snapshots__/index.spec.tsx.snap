// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render images should render images 1`] = `
<div>
  <img
    alt="Photo of user"
    src="https://reqres.in/img/faces/7-image.jpg"
  />
</div>
`;
